use_cases/community-manager.md

```diff
@@ -0,0 +1,220 @@
+# Community Manager
+
+## Description
+
+This use case deals with how a project's community manager might keep track of a collection of repositories. 
+
+## Main goal
+
+Help a community manager maintain awareness of a repository collection. 
+
+Support and enhance communication about a repository collection to other stakeholders. In some cases these will be individuals who are funding the community manager's role. In other cases, and possibly simultaneously, they will be organizations with a shared interest in a particular community. 
+
+## Primary Actor
+An open source project(s) community manager. This individual or group of individuals will be responsible for one or more repositories connected together as a project. 
+
+## Stakeholders
+
+* Community manager: Responsible for developing a sense of community in the project. 
+* Contributors: Interested in contributed to projects that have some apparent indications of health. They want to invest time wisely. 
+* Organizations: This could be a company or companies, a foundation, academic unit or collection of cooperative developers who are, together, looking to ensure the health and sustainability of a community. 
+* Project community in general, for self-awareness about how the onboarding
+process is performing.
+* Prospective contributors, to make decisions on whether to
+invest their time and effort in joining a community,
+or in general, produce contributions for the project.
+
+## Vocabulary
+* Community Manager: takes a variety of perspectives, depending on where their communities are in the lifecycle of growth, maturity and decline. 
+
+* Community: A collection of interconnected repositories. Sometimes referred to as an "ecosystem" if it scales largely enough. 
+
+* Community Manager Metrics: This is an evolving use case of what we are learning from community managers, some of whom we are working with on live experiments with a CHAOSS project prototyping software tool called Augur (http://www.github.com/CHAOSS/augur). At this point we are paying particular focus to how community managers consume metrics and how the presentation of open source software health and sustainability metrics could make them more and in some cases less useful for doing their jobs. 
+
+* Others as defined in [First Code Contribution](./first-code-contributions.md) use case. 
+
+
+## Preconditions
+
+* The community manager should have an interest in a broad perspective on many projects. 
+* The community manager should have a larger than 1 set of repositories they are interested in tracking collectively.  
+* A community must exist
+
+## Questions
+
+Date Summarized Comparison Metrics
```

Comment:
i like the way you express the questions by example. I still think that maybe we could just frame the general idea as the question (eg. "How do n projects compare, according to some date-summarized metrcis?"), and then in the explanation mention these as examples of specific questions that could be solved.

But I'm also happy if we keep it as it is for now, and refine all this later, in a further pull request.

Reply:
You mean a ranking of projects based on metrics x, y, z in time frame (month, quarter, year, custom date)?

Reply:
Yes, @rsanchez-seerene that could be a way of implementing it. But in the text in the use case, I would express it in a more general way, and also a way that does not imply ranking, but just comparison.